### PR TITLE
Hardcoded links have been changed to dynamic urls

### PR DESCRIPTION
**Pull request by ephemeralshay**
## 🛠️ Fixes Issue
Used template inheritance to change hard-coded link in Navbar and updated Index, Features, Pricing, Login and Contact Us pages accordingly.

### CODE REVIEW

File: templates/home/Features/features.html [hunk 1]
```diff
@@ -1,140 +1,52 @@
...
+<body>
 
-      <h2 class="fea">      In-depth concept coverage</h2>
-      <h2  class="fea">      One step solution</h2>
-      <h2  class="fea">      24*7 Doubt solving</h2>
-      <h2  class="fea">      1 on 1 Mentoring</h2>
-      <h2  class="fea">      Project Guidance</h2>
-      <h2  class="fea">      Certification Included</h2>
-      <br><br>
+  {% include 'includes/navbar.html' %}
```
Comment (KKhushhalR2405) [inline templates/home/Features/features.html:22]: This include is not required. Inherit directly from base.html. For reference [base.html](https://github.com/arpit456jain/CodingEasy/blob/master/templates/home/base.html) and [button.html](https://github.com/arpit456jain/CodingEasy/blob/master/templates/home/bootstrap/button/button.html). Including navbar.html in every template does not make any sense. The same goes for footer.

**Review by KKhushhalR2405 — CHANGES_REQUESTED** [review]
Kindly consider the review given above. It has to be followed for all the files

File: templates/home/Features/features.html (hunk 1 above)
Comment (ephemeralshay) [reply templates/home/Features/features.html:22]: Alright. Will try to resolve the matter right away.

**Comment by ephemeralshay** [comment]
All the templates (index/feature/pricing and all course files) except Login and Contact have been updated with template inheritance from base.html

**Comment by ephemeralshay** [comment]
@arpit456jain @KKhushhalR2405 Please review and let me know

**Comment by raghavTinker** [comment]
Seems good to me @KKhushhalR2405 please give the final approval

**Review by arpit456jain — APPROVED** [review]

**Pull request merged**